spec/models/user_spec.rb

```diff
@@ -4,8 +4,15 @@
 
   subject { create(:user) }
 
-  it 'validates presence of first_name'
-  it 'validates presence of last_name'
+  it 'validates presence of first_name' do
+    user = build(:user, first_name: nil)
```

Comment:
As shoulda is more robust and definitely checks specific field. As record in that scenario could be invalid by other means